src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -100,4 +100,33 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
             return response;
         }
     }
+    
+    // Custom web hooks for extensions. 
+    public class HookController : ApiController
+    {
+        private readonly WebScriptHostManager _scriptHostManager;
+
+        public HookController(WebScriptHostManager scriptHostManager)
+        {
+            _scriptHostManager = scriptHostManager;
+        }
+                
+        [Route("hook/{name}")]
+        [HttpGet]
```

Comment:
Why do we need Get? Is this because this is for more than just traditional webhooks (which generally are POST only)?